### ISSUE TITLE
Request for a new release with the current working MATSim repository and newer jdk version

### ISSUE THREAD

**Issue by auzpatwary37** [matsim-org/matsim-episim-libs#100]
Hi, the latest release cannot resolve dependencies with the error "Could not transfer artifact org.matsim.contrib:accessibility:pom:13.0 from/to matsim (https://repo.matsim.org/repository/matsim/): The operation was cancelled." 

I think this has to do with the recent change in MATSim repo and the snapshot version.

**Comment by rakow** [comment]
The pom already contains the updated repositories.  All recent builds on Github are working and are also using the 13.0 version.
The error message looks more like a server problem or temporary connection problem to me. Maybe just try it again.
In any case, we might think about upgrading to a recent MATSim version.
